readme.md

```diff
@@ -50,7 +50,8 @@
   - [Amine Hakkou](https://www.hakkou.me/) - Javascript, React Native ....
   - [Abderrahim SOUBAI-ELIDRISI](https://www.soubai.me/) - Javascript, Node.js , Angular ....
   - [Meriem Zaid](https://imeriem.com/) - Javascript, Python, React ....
-
+  - [Anas El Alami](https://beta.anaselalami.com/) - Javascript, NodeJS, Vuejs, ...
```

Comment:
Can you please add your blog URL instead of your portfolio?

Reply:
Unfortunately i don't have a blog for the moment, I'll remove my portfolio link  👍

Reply:
sorry for that, I think it's time to start 😄

Reply:
yeah, i have only two medium posts for now, at least i can start adding them as a fresh `restart` 👍 , thanks